go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/rivo/uniseg v0.4.4 h1:8TfxU8dW6PdqD27gjM8MVNuicgxIjxpm4K7x4jp8sis=
+github.com/rivo/uniseg v0.4.4/go.mod h1:FN3SvrM+Zdj16jyLfmOkMNblXMcoc8DfTHruCPUcx88=
```

Comment:
Updated PR to attempt to use a direct implementation from the uniseg code but there's an awful lot of lookup tables that are required to implement them, and the golang.org/x/text issue does not seem to have progressed since 2019.